### ISSUE TITLE
Difference in behavior depending on previous animation

### ISSUE THREAD

**Issue by marick** [mdgriffith/elm-style-animation#26]
TL;DR: An animation that flashes two different colors works only if a different animation precedes it. I suspect I'm doing something wrong, but it might be a bug.

-----

I have an animation of an IV drip. The drip has two different appearances, depending on the speed the user types in. If the speed is slow, the drops look like this:

![slow](https://cloud.githubusercontent.com/assets/71909/19537108/6670416e-9614-11e6-9388-758744c94e09.gif)

(The animation is started or changed when focus leaves the text field.)

If the speed is fast, I show a bar that varies between two shades of grey to give a (crude) flowing appearance:

![fast](https://cloud.githubusercontent.com/assets/71909/19537273/f61096a2-9614-11e6-8aa6-59c7d9e7af39.gif)

However, that only happens if you switch from the slow animation to the fast one (by first typing something like "2" in the text field, leaving it, going back, then changing the value to "20"). 

* high-speed changed to a different high speed still shows the solid bar.
* fast->slow->fast shows the pulsing bar on the second fast.
* Indeed, once there's been a slow, any future fast speed will pulse.

The source can be found here: https://github.com/marick/eecrit/blob/looping-oddity/web/elm/IV/Apparatus/Droplet.elm You can try the animation yourself at https://new.critter4us.com/iv (though I'm changing this frequently). 

Both animations are started by this:

```elm
changeDropRate dropsPerSecond animation =
  let
    loop = Animation.loop (animationSteps dropsPerSecond)
  in
    Animation.interrupt [loop] animation
```

The steady stream is this:

```elm
steadyStream = 
  [ Animation.toWith (easing fallingTime) View.streamState1
  , Animation.toWith (easing fallingTime) View.streamState2
  ]
```

The "stream states" are from https://github.com/marick/eecrit/blob/looping-oddity/web/elm/IV/Apparatus/DropletView.elm

```elm
streamState1 =
  [ Animation.points flowInChamber
  , Animation.fill Palette.aluminum
  ]    
      
streamState2 =
  [ Animation.points flowInChamber
  , Animation.fill Palette.shiftedAluminum
  ]    
```

For what it's worth, the falling drop is done like this:

```elm
    [ Animation.set View.missingDrop
    , Animation.toWith (easing hangingTime) View.hangingDrop
    , Animation.toWith (easing fallingTime) View.fallenDrop
    ]
```

------------

I'm using 3.2.0, but it seems the same issue happens on 3.0 and 3.1.

Behavior is the same on Chrome and Safari.

**Comment by mdgriffith** [comment]
If you can create an [SSCCE](http://sscce.org/) that shows an issue/bug with `elm-style-animation`,  I'll definitely look into it.  Otherwise try posting to elm-discuss or the [elm slack](https://elmlang.herokuapp.com/) for general animation help in your application.

Good luck!
